### PR TITLE
ref(tsc): utils to tsx

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: static/app/components/organizations/pageFilters/container.spec.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import OrganizationsStore from 'sentry/stores/organizationsStore';
 import OrganizationStore from 'sentry/stores/organizationStore';
 import PageFiltersStore from 'sentry/stores/pageFiltersStore';
 import ProjectsStore from 'sentry/stores/projectsStore';
-import {getItem, setItem} from 'sentry/utils/localStorage';
```
Comment (JonasBa) [inline static/app/components/organizations/pageFilters/container.spec.tsx:11]: cc @vuluongj20, there seemed to have been a lot of type errors in this file

File: static/app/components/projects/useProjectCreationAccess.spec.tsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
-const {useProjectCreationAccess} = require('./useProjectCreationAccess');
```
Comment (ryan953) [inline static/app/components/projects/useProjectCreationAccess.spec.tsx:1]: woah, weird to see this

Comment (JonasBa) [reply static/app/components/projects/useProjectCreationAccess.spec.tsx:1]: Yeah, I was equally surprised :D

**Review by ryan953 — APPROVED** [review]

File: static/app/components/search/sources/formSource.spec.tsx [hunk 3]
```diff
@@ -35,7 +37,7 @@ describe('FormSource', function () {
   });
 
   afterEach(function () {
-    ActionCreators.loadSearchMap.mockRestore();
+    (ActionCreators.loadSearchMap as jest.Mock).mockRestore();
```
Comment (evanpurkhiser) [inline static/app/components/search/sources/formSource.spec.tsx:40]: I've just instead been switching this to `Jest.restoreAllMocks()`

File: static/app/components/projects/useProjectCreationAccess.spec.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/components/projects/useProjectCreationAccess.spec.tsx:1]: Are there any more in the codebase we can get rid of?

Comment (JonasBa) [reply static/app/components/projects/useProjectCreationAccess.spec.tsx:1]: No idea, trying to see as I go :D

File: static/app/components/search/sources/formSource.spec.tsx (hunk 3 above)
Comment (JonasBa) [reply static/app/components/search/sources/formSource.spec.tsx:40]: Ah nice, we should do that indeed. I'll make a note for myself to change this next time, the casts are 😞

File: static/app/components/organizations/pageFilters/container.spec.tsx (hunk 1 above)
Comment (vuluongj20) [reply static/app/components/organizations/pageFilters/container.spec.tsx:11]: (Didn't write this file but) Thanks for fixing them!

**Pull request merged**